src/images/icons/rotate_90.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i see that it's not cleared at all from unnecessary markup, maybe u could pass it through https://jakearchibald.github.io/svgomg/
that's the good practise we try to keep.
u can take a look at other icons how "clean" they are